nexus-sdk/src/main/java/io/nexusrpc/handler/ServiceHandler.java

```diff
@@ -87,7 +103,7 @@ public HandlerResultContent fetchOperationResult(
     if (handler == null) {
       throw newUnrecognizedOperationException(context.getService(), context.getOperation());
     }
-    Object result = handler.fetchResult(context, details);
+    Object result = getOperationHandler(handler).fetchResult(context, details);
```

Comment:
If we did https://github.com/nexus-rpc/sdk-java/pull/14/files#r1855645828 we could also consider switching where we call `getOperationHandler` to be before the data coversion. This would allow to fail requests before the data converter runs.

Reply:
Yeah, that'd be great.

Reply:
Done, this only applied to `start`